earthkit/data/sources/cds.py

```diff
@@ -123,27 +123,5 @@ def requests(self, **kwargs):
 
         return result
 
-    def to_pandas(self, **kwargs):
-        pandas_read_csv_kwargs = dict(
-            comment="#",
-            parse_dates=["report_timestamp"],
```

Comment:
`parse_dates` are not added as an argument to csv.to_pandas()

Reply:
test added to test_cds.py: `test_cds_non_observation_csv_file`